utils/payment-actions.js

```diff
@@ -42,13 +42,15 @@ export function fillCreditCardInfo(
 
   queryIframe($iframe => {
     const $body = getIframeBody($iframe)
+
+    // We type with force:true because of https://github.com/cypress-io/cypress/issues/5830
     cy.wrap($body)
       .find(`#creditCardpayment-card-${options.id || '0'}Number`)
-      .type('4040240009008936')
+      .type('4040240009008936', { force: true })
```

Comment:
Is there any specific reason to use `force` instead of using `blur`(as mentioned [here](https://github.com/cypress-io/cypress/issues/5830#issuecomment-570638375))?

Reply:
The only reason is that `blur` didn't work 😐